diff --git b/packages/openapi/lib/model/get_supply_chain200_response_data.dart a/packages/openapi/lib/model/get_supply_chain200_response_data.dart
index 557f3aa2..ccf66335 100644
--- b/packages/openapi/lib/model/get_supply_chain200_response_data.dart
+++ a/packages/openapi/lib/model/get_supply_chain200_response_data.dart
@@ -59,10 +59,15 @@ class GetSupplyChain200ResponseData {
         return true;
       }());
 
+      final map = json[r'exportToImportMap'] == null
+          ? const <String, List<String>>{}
+          : mapCastOfType<String, List>(json, r'exportToImportMap')!;
+      final casted = Map<String, List<String>>.fromIterable(
+        map.keys,
+        value: (key) => map[key]!.cast<String>(),
+      );
       return GetSupplyChain200ResponseData(
-        exportToImportMap: json[r'exportToImportMap'] == null
-            ? const {}
-            : mapCastOfType<String, List>(json, r'exportToImportMap'),
+        exportToImportMap: casted,
       );
     }
     return null;
